src/library-authoring/data/api.ts

```diff
@@ -86,6 +96,27 @@ export interface ContentLibrary {
   updated: string | null;
 }
 
+export interface LibraryTeamMember {
+  username: string;
+  email: string;
+  accessLevel: string;
```

Comment:
Could we define this as a `LibraryRole` type instead? Or at least `'admin' | 'author' | 'read'` instead of `string`.